README.md

```diff
@@ -166,35 +166,7 @@ specification:
       valid requests make it up until here. You can also specify an own function resolver function in order to hook into
       your own framework.
 
-## Development on swagger1st
```

Comment:
@sarnowski Any other comments on the PR? As for my questions: they're aimed at helping non-Zalandos to understand why they should use this project, and clarifying ambiguities for current/potential users.